.github/workflows/check.yml

```diff
@@ -112,6 +112,19 @@ jobs:
       - name: Run tests and determine coverage
         run: cargo +${{ matrix.rust-toolchain }} llvm-cov nextest --features ci --all-targets --no-fail-fast --lcov --output-path lcov.info
 
+      - name: Run client/server transfer
+        run: |
+          cargo +${{ matrix.rust-toolchain }} build --bin neqo-client --bin neqo-server
+          cargo +${{ matrix.rust-toolchain }} run --bin neqo-server -- $HOST:4433 &
+          PID=$!
+          cargo +${{ matrix.rust-toolchain }} run --bin neqo-client -- --output-dir . https://$HOST:4433/$SIZE
+          kill $PID
+          [ "$(wc -c <"$SIZE")" -eq "$SIZE" ] || exit 1
```

Comment:
Oh, this statement is weird...  The fact that the filename is "$SIZE" is offputting.

Reply:
Yeah, there doesn't seem to be a way to choose a name for downloaded objects. I could obviously have two env variables `FILENAME` and `SIZE` that are both `54321`, but I doubt that's much clearer.